packages/ui/src/ii-login-button.ts

```diff
@@ -0,0 +1,260 @@
+  }
+}
+
+// #region Logo
+const iiLogo = `<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" preserveAspectRatio="xMidYMid meet" viewBox="-1 -1 107.42470599999999 54.6597" width="103.42" height="50.66" class="w-8 h-8"><defs><path id="a2oTCciHwU" d="M59.62 9.14C56.48 11.99 53.72 15.07 51.71 17.53C51.71 17.53 54.95 21.18 58.49 25.09C60.41 22.72 63.17 19.51 66.36 16.61C72.26 11.2 76.1 10.11 78.33 10.11C86.64 10.11 93.37 16.92 93.37 25.35C93.37 33.7 86.64 40.51 78.33 40.6C77.94 40.6 77.46 40.55 76.89 40.42C79.29 41.52 81.92 42.31 84.37 42.31C99.58 42.31 102.55 32.03 102.72 31.28C103.16 29.39 103.42 27.42 103.42 25.4C103.38 11.38 92.15 0 78.33 0C72.56 0 66.27 3.08 59.62 9.14Z"></path><linearGradient id="gradientc2rIvyDhN" gradientUnits="userSpaceOnUse" x1="65.17" y1="3.67" x2="99.9" y2="39.44"><stop style="stop-color:#f15a24;stop-opacity:1;" offset="21%"></stop><stop style="stop-color:#fbb03b;stop-opacity:1;" offset="68.41000000000001%"></stop></linearGradient><path id="aFWcwFIax" d="M43.8 41.52C46.95 38.66 49.7 35.59 51.71 33.13C51.71 33.13 48.48 29.48 44.94 25.57C43.01 27.94 40.26 31.15 37.07 34.05C31.17 39.41 27.28 40.55 25.09 40.55C16.79 40.55 10.05 33.74 10.05 25.31C10.05 16.96 16.79 10.15 25.09 10.06C25.48 10.06 25.97 10.11 26.53 10.24C24.13 9.14 21.51 8.35 19.06 8.35C3.85 8.35 0.87 18.63 0.7 19.38C0.26 21.27 0 23.24 0 25.26C0 39.28 11.23 50.66 25.09 50.66C30.86 50.66 37.16 47.58 43.8 41.52Z"></path><linearGradient id="gradientaIMror5Zz" gradientUnits="userSpaceOnUse" x1="38.22" y1="46.97" x2="3.5" y2="11.2"><stop style="stop-color:#ed1e79;stop-opacity:1;" offset="21%"></stop><stop style="stop-color:#522785;stop-opacity:1;" offset="89.29%"></stop></linearGradient><path id="e1RZG6bOc" d="M19.1 8.57C5.24 8.23 1.16 18.1 0.81 19.37C3.47 8.3 13.37 0.04 25.14 0C34.73 0 44.43 9.21 51.59 17.53C51.6 17.51 51.61 17.5 51.62 17.49C51.62 17.49 54.86 21.13 58.4 25.04C58.4 25.04 62.42 29.7 66.71 33.79C68.37 35.37 76.45 41.74 84.23 41.96C98.49 42.36 102.42 31.94 102.64 31.15C100.02 42.27 90.09 50.57 78.29 50.62C68.69 50.62 58.99 41.4 51.8 33.08C51.78 33.1 51.77 33.11 51.76 33.13C51.76 33.13 48.52 29.48 44.98 25.57C44.98 25.57 40.96 20.91 36.68 16.83C35.01 15.25 26.88 8.79 19.1 8.57ZM0.81 19.37C0.8 19.4 0.79 19.43 0.79 19.46C0.79 19.44 0.8 19.41 0.81 19.37Z"></path></defs><g><g><use xlink:href="#a2oTCciHwU" opacity="1" fill="url(#gradientc2rIvyDhN)"></use><g><use xlink:href="#a2oTCciHwU" opacity="1" fill-opacity="0" stroke="#000000" stroke-width="1" stroke-opacity="0"></use></g></g><g><use xlink:href="#aFWcwFIax" opacity="1" fill="url(#gradientaIMror5Zz)"></use><g><use xlink:href="#aFWcwFIax" opacity="1" fill-opacity="0" stroke="#000000" stroke-width="1" stroke-opacity="0"></use></g></g><g><use xlink:href="#e1RZG6bOc" opacity="1" fill="#29abe2" fill-opacity="1"></use><g><use xlink:href="#e1RZG6bOc" opacity="1" fill-opacity="0" stroke="#000000" stroke-width="1" stroke-opacity="0"></use></g></g></g></svg>`;
```

Comment:
This can be optimized, I got this from https://jakearchibald.github.io/svgomg/

```
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" class="w-8 h-8" viewBox="-1 -1 107.425 54.66"><defs><path id="a" d="M59.62 9.14c-3.14 2.85-5.9 5.93-7.91 8.39 0 0 3.24 3.65 6.78 7.56 1.92-2.37 4.68-5.58 7.87-8.48 5.9-5.41 9.74-6.5 11.97-6.5 8.31 0 15.04 6.81 15.04 15.24 0 8.35-6.73 15.16-15.04 15.25-.39 0-.87-.05-1.44-.18 2.4 1.1 5.03 1.89 7.48 1.89 15.21 0 18.18-10.28 18.35-11.03.44-1.89.7-3.86.7-5.88C103.38 11.38 92.15 0 78.33 0c-5.77 0-12.06 3.08-18.71 9.14Z"/><path id="c" d="M43.8 41.52c3.15-2.86 5.9-5.93 7.91-8.39 0 0-3.23-3.65-6.77-7.56-1.93 2.37-4.68 5.58-7.87 8.48-5.9 5.36-9.79 6.5-11.98 6.5-8.3 0-15.04-6.81-15.04-15.24 0-8.35 6.74-15.16 15.04-15.25.39 0 .88.05 1.44.18-2.4-1.1-5.02-1.89-7.47-1.89C3.85 8.35.87 18.63.7 19.38c-.44 1.89-.7 3.86-.7 5.88 0 14.02 11.23 25.4 25.09 25.4 5.77 0 12.07-3.08 18.71-9.14Z"/><path id="e" d="M19.1 8.57C5.24 8.23 1.16 18.1.81 19.37 3.47 8.3 13.37.04 25.14 0c9.59 0 19.29 9.21 26.45 17.53.01-.02.02-.03.03-.04 0 0 3.24 3.64 6.78 7.55 0 0 4.02 4.66 8.31 8.75 1.66 1.58 9.74 7.95 17.52 8.17 14.26.4 18.19-10.02 18.41-10.81-2.62 11.12-12.55 19.42-24.35 19.47-9.6 0-19.3-9.22-26.49-17.54-.02.02-.03.03-.04.05 0 0-3.24-3.65-6.78-7.56 0 0-4.02-4.66-8.3-8.74-1.67-1.58-9.8-8.04-17.58-8.26ZM.81 19.37c-.01.03-.02.06-.02.09 0-.02.01-.05.02-.09Z"/><linearGradient id="b" x1="65.17" x2="99.9" y1="3.67" y2="39.44" gradientUnits="userSpaceOnUse"><stop offset="21%" stop-color="#f15a24"/><stop offset="68.41%" stop-color="#fbb03b"/></linearGradient><linearGradient id="d" x1="38.22" x2="3.5" y1="46.97" y2="11.2" gradientUnits="userSpaceOnUse"><stop offset="21%" stop-color="#ed1e79"/><stop offset="89.29%" stop-color="#522785"/></linearGradient></defs><use xlink:href="#a" fill="url(#b)"/><use xlink:href="#a" fill-opacity="0" stroke="#000" stroke-opacity="0"/><use xlink:href="#c" fill="url(#d)"/><use xlink:href="#c" fill-opacity="0" stroke="#000" stroke-opacity="0"/><g><use xlink:href="#e" fill="#29abe2"/><use xlink:href="#e" fill-opacity="0" stroke="#000" stroke-opacity="0"/></g></svg>
```

Reply:
thanks!